.taskcluster.yml

```diff
@@ -181,7 +180,7 @@ tasks:
                         git fetch ${repository} refs/tags/${head_rev}
                         && git config advice.detachedHead false
                         && git checkout FETCH_HEAD
-                        && python automation/taskcluster/decision_task.py github-release ${event.release.tag_name}
+                        && python automation/taskcluster/decision_task.py github-release ${event.release.tag_name} --staging
```

Comment:
Nit: Let's make this flag dynamic too

Reply:
Removed this change from my PR, sorry for bad comms :)